cfbs/index.py

```diff
@@ -98,6 +101,10 @@ def data(self) -> dict:
             self._expand_index()
         return self._data
 
+    @property
+    def url(self) -> str:
+        return self._unexpanded
+
```

Comment:
The reason why this attribute is not called URL is because it can also be just a dict with the data. See the logic in `_expand_index()`.

So, I don't agree with this change, you can either change the internals to have 2 different variables: unexpanded `url` and `data` (both of which can be None), or you can change the `url` method to return `None` if `self._unexpanded` is dict:

```suggestion
    @property
    def url(self) -> str:
        if type(self._unexpanded) is not str:
            return None
        return self._unexpanded

```

Reply:
Yeah, makes sense. Made a change.